### ISSUE TITLE
Navigation - mobile

### ISSUE THREAD

**Issue by martoio** [RICHLAX/f1-3-c2p1-colmar-academy#1]
It would be easier if you make a separate div that has all of the elements needed for the mobile navigation. (Logo + the other 3 images). Then you set the flex parameters to make them space out evenly. In the media query you set ``display: none;`` (IMPORTANT! not ``visibility: hidden;``) on the non-mobile navigation. That way you avoid the weird spacing issues

**Comment by roninlax** [comment]
yes thank you for the idea...It was bothering me that the logo and the nav elements were not spacing out evenly...you gave me the solution!
